src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -72,9 +72,14 @@ object Import {
       """Declares endpoints. The default is Map("web" -> Endpoint("http", 0, Set("http://:9000"))) where the service name is the name of this project. The "web" key is used to form a set of environment variables for your components. For example you will have a `WEB_BIND_PORT` in this example."""
     )
 
+    val checkInitialDelay = SettingKey[Int](
```

Comment:
Another trade off is that the user need to import `scala.concurrent.duration._` in order to specify in the build.sbt:
```
BundleKeys.checkInitialDelay := 10.seconds
```

Also query params in the `BundleKeys.checks` command use `Int` as type to specify things in seconds. Using `FiniteDuration` as type for `BundleKeys.checkInitialDelay` would like this:
```
import scala.concurrent.duration._
BundleKeys.checkInitialDelay := 10.seconds
BundleKeys.checks := Seq(uri("$WEB_HOST?retry-count=5&retry-delay=3"))
```

With `Int` it would like this:
```
BundleKeys.checkInitialDelay := 10
BundleKeys.checks := Seq(uri("$WEB_HOST?retry-count=5&retry-delay=3"))
```

@huntc I think the latter with `Int` as type is more intuitiv for the user without looking up the documentation which type need to be specified. WDYT?